sort.py

```diff
@@ -0,0 +1,30 @@
+import csv
+
+with open('db.csv') as csv_file:
```

Comment:
the file name is hardcoded. it will be better if the input file can be passed as an argument to the script.

Reply:
I made it into a function with the file as the argument. Is that what you wanted?

Reply:
Yes somewhat like this. I will check in detail later and let you know.